src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,45 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
The method name `getStatistic` does not fully convey what the method does. Consider renaming it to something more descriptive, such as `generateReport` or `calculateAndWriteStatistics`, to better reflect its purpose following the method naming convention.